src/app/(sidebar)/transaction/sign/components/Overview.tsx

```diff
@@ -284,6 +285,7 @@ export const Overview = () => {
                 onUpdate={onUpdateSecretInputs}
                 validate={validate.secretKey}
                 placeholder="Secret key (starting with S) or hash preimage (in hex)"
+                autocomplete="off"
```

Comment:
We need to make sure we never save private keys in the browser with autocomplete.